src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -693,7 +693,7 @@ public String getCsvHeader() {
 
     @Override
     public String getQuestionTypeChoiceOption() {
-        return "<option value = \"CONTRIB\">" + Const.FeedbackQuestionTypeNames.CONTRIB + "</option>";
+        return "<li data-questionType = \"CONTRIB\"><a>" + Const.FeedbackQuestionTypeNames.CONTRIB + "</a></li>";
```

Comment:
`questionType` is shown here, but in the test files, they show up as `questiontype`. I think we can go with `data-questiontype` here instead, to be consistent with the test files